AlembicImporter/Assets/UnitTests/AlembicExportTest.cs

```diff
@@ -231,5 +241,358 @@ public IEnumerator TestClothExport()
 
             TestAbcImported(exportPath);
         }
+        //Create And Delete
+        [UnityTest]
+        public IEnumerator TestCreateAndDelete()
+        {
+            // open scene
+            SceneManagement.EditorSceneManager.LoadScene(cd_scene, UnityEngine.SceneManagement.LoadSceneMode.Single);
+
+            // yield once while scene loads
+            yield return null;
+
+            yield return new WaitForSeconds(9f);
+            //  EditorApplication.isPlaying = false;
+            yield return null;
+            TestAbcImported("Assets/UnitTests/RecorderUnitTests/Recorder.abc");
+        }
+        //Test Export
+        [UnityTest]
+        public IEnumerator TestExport()
```

Comment:
this test and the tests below all seem to use the same code. Can you create a helper function(s) that does most of the work and have these call it?

e.g.
```
private IEnumerator AlembicExportTest(string scene){
    // open scene
    SceneManagement.EditorSceneManager.LoadScene(scene, UnityEngine.SceneManagement.LoadSceneMode.Single);

     // yield once while scene loads
     yield return null;

     ...
}
```